.github/workflows/swift.yaml

```diff
@@ -0,0 +1,106 @@
+name: Swift CI
+
+on:
+  push:
+    branches:
+      - master
+      - '!gh-pages'
+    paths:
+      - '*.podspec'
+      - 'Gemfile*'
+      - 'Package.swift'
+      - 'swift/**'
+      - '.github/workflows/swift.yaml'
+  pull_request:
+    paths:
+      - '*.podspec'
+      - 'Gemfile*'
+      - 'Package.swift'
+      - 'swift/**'
+      - '.github/workflows/swift.yaml'
+
+jobs:
+  development-apps:
+    runs-on: macOS-latest
+
+    strategy:
+      matrix:
+        scheme: 
+        - Development-Unit-WorkflowTests
+        - Development-Unit-WorkflowUITests
+        - Development-Unit-SplitScreenTests
+        - Development-Unit-TicTacToeTests
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Install Bundler
+      run: |
+        gem update --system
+        gem install bundler
+
+    - name: Bundle Install
+      run: bundle install
+
+    - name: Pod Install
+      run: |
+        bundle exec pod gen Development.podspec
+
+    - name: Switch Xcode
+      run: sudo xcode-select -s /Applications/Xcode_11.3.1.app
+
+    - name: Build & Test
+      run: |
+        set -o pipefail && xcodebuild -workspace gen/Development/Development.xcworkspace -scheme ${{ matrix.scheme }} -destination platform\=iOS\ Simulator,OS\=13.3,name\=iPad\ Pro\ \(9.7-inch\) build test | xcpretty
+
+  spm:
+    runs-on: macOS-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Swift Package Manager - iOS
+      run: |
+        xcodebuild -scheme "Workflow-Package" test -destination "name=iPhone 11"
+
+    - name: Swift Package Manager - macOS
+      run: |
+        xcodebuild -scheme "Workflow-Package" test
+
+  tutorial:
+    runs-on: macOS-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Install Bundler
+      run: |
+        gem update --system
+        gem install bundler
+
+    - name: Bundle Install
+      run: bundle install
+
+    - name: Tutorial App
+      run: |
+        cd swift/Samples/Tutorial
+        bundle exec pod repo update
```

Comment:
if this is taking a while the podfile could be updated to use `https://cdn.cocoapods.org` and remove the call to `pod repo update`